### ISSUE TITLE
Command `checkly run` does not work at all

### ISSUE THREAD

**Issue by tnolet** [checkly/checkly-cli#137]
I cannot run any of the checks in my account. It always returns `ERROR   No check found with that name, please try again`

```
timnolet@Tims-MacBook-Pro-2 analytics-cdn-proxy % checkly checks


name              checkType  frequency  locations               activated
----------------  ---------  ---------  ----------------------  ---------
API Check #1      API        5          eu-central-1,eu-west-2  true     
Browser Check #4  BROWSER    10         eu-central-1,eu-west-2  true     
bonohealth        BROWSER    5          eu-central-1,eu-west-2  true     
Ryan Florence     BROWSER    720        eu-central-1,eu-west-2  true     

timnolet@Tims-MacBook-Pro-2 analytics-cdn-proxy % checkly run bonohealth


 ERROR   No check found with that name, please try again.

timnolet@Tims-MacBook-Pro-2 analytics-cdn-proxy % 
````

**Comment by umutuzgur** [comment]
@tnolet this is working for me. I wonder why it doesn't work for you

**Comment by brampling** [comment]
@umutuzgur Tim is running `checkly checks` then trying to run one of those using `checkly run`. The problem is that checkly checks is listing cloud-based checks while checkly run is trying to run a check from the local directory, not from the cloud. Both commands are working as built, but not meeting the expectation of the user. It is counterintuitive to use checkly checks to see a list of checks that I can't run using checkly run. The cloud vs. local paradigm is not clear here.

**Comment by ianaya89** [comment]
I get the point, I think `checkly run` should be cloud/local agnostic and allow users to run both types of checks using `checkId` or `checkName` ar argument

**Comment by umutuzgur** [comment]
@ianaya89 @brampling I think it would be nice the brainstorm on this one. I would make this command run initially just the local resources. I can't think of a use case for running checks that are already deployed, unless we replace deployment triggers which should be under a different command in my opinion.

**Comment by ianaya89** [comment]
@umutuzgur I think makes sense, maybe `run` is not the proper name for this command and should actually be something like `test` or `try`. The motivation of this command is to test your local stuff before deploying to Checkly

**Comment by tnolet** [comment]
> @umutuzgur I think makes sense, maybe `run` is not the proper name for this command and should actually be something like `test` or `try`. The motivation of this command is to test your local stuff before deploying to Checkly

I don't agree. Using `run` is the most simple verb. As a user, I have no idea what `test` or `try` means. You "run" a test, you "run" some code: similarly you "run" a check.

Whether that check exists on the cloud, or only in my codebase (not yet deployed) is irrelevant. I see a check, I want to run it.

**Comment by tnolet** [comment]
> @ianaya89 @brampling I think it would be nice the brainstorm on this one. I would make this command run initially just the local resources. I can't think of a use case for running checks that are already deployed, unless we replace deployment triggers which should be under a different command in my opinion.

@umutuzgur I don't think we need to replace deployment triggers. We can just introduce a new (async?) API endpoint that triggers checks that are already defined in the cloud, or take a payload of and run that payload: this run would be ephemeral of course as there is not state to persist.
